### ISSUE TITLE
Inconsistent compatibility versioning + outdated README

### ISSUE THREAD

**Issue by afx114** [elastic/elasticsearch-py#276]
The README needs to be updated, because it currently states the following version compatibility:

```
# Elasticsearch 1.0
elasticsearch>=1.0.0,<2.0.0
```

But that is not the case.  I recently installed 1.7.0 and it broke our calls to `put_mapping` because `ignore_conflicts` was removed as a parameter for 2.0 compatibility in commit 36d8d4b545938fd37988b72ccb8e19b58fe62882.

Perhaps the ES2.0 compatible versions of elasticsearch-py should start with v2.0.0, and leave <2.0.0 for ES1.0?  Ideally it'd look something like this:

```
# Elasticsearch 2.0
elasticsearch>=2.0.0,<3.0.0

# Elasticsearch 1.0
elasticsearch>=1.0.0,<2.0.0
```

**Comment by HonzaKral** [comment]
Yes, this is exactly the plan! I will work n it soon, I hope for next week.

Thanks for raising the issue!

**Comment by HonzaKral** [comment]
README updated 1.8.0 released and 2.0.0 is on it's way.

thanks again
